src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -25,6 +30,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
         // TODO: add log message about successful fetched data from DB
+        logger.info("List of products fetched from DB successfully.");
```

Comment:
The log message states that the list of products was fetched from the DB successfully, but in reality, this method returns dummy data. Consider updating the log message to reflect that this is a mock implementation or test data.